tailwind.config.ts

```diff
@@ -26,6 +26,10 @@ const config = {
         secondary: "#FFCB09",
         error: "#DC143C",
         success: "#17C964",
+        NeoRed: "#e24a57",
+        NeoYellow: "#fba834",
+        NeoBlue: "#396eeb",
+        NeoGreen: "#25b195",
```

Comment:
개발 컨벤션에 맞춰서 모두 camelCase 로 변경해주시면 감사하겠습니다.
```suggestion
        neoRed: "#e24a57",
        neoYellow: "#fba834",
        neoBlue: "#396eeb",
        neoGreen: "#25b195",
```